src/components/BackButton.vue

```diff
@@ -1,7 +1,7 @@
 <template>
```

Comment:
Please note that my approval for this PR is limited to the changes to the back button (https://github.com/pascalwengerter/musclemate-frontend/pull/32/commits lists three individual commit, I'd ask you to remove the other two commits from this branch ;) )

Reply:
Got it, I'll remove the two commits that are not related to the changes made to the back button, as per your request.